syfertext/doc.py

```diff
@@ -338,11 +338,8 @@ def get_token_vectors(self, excluded_tokens: Dict[str, Set[object]] = None) -> t
             if include_token:
                 token_vectors.append(token.vector)
 
-        # # Convert to Numpy array.
-        # token_vectors = np.array(token_vectors)
-
         # Convert to torch tensor
-        token_vectors = torch.tensor(token_vectors, dtype=torch.float32)
+        token_vectors = torch.stack(tensors = token_vectors)
```

Comment:
@AlanAboudib  I was thinking maybe instead of appending to a list, just stack the tensors in the previous step? Probably will save some memory.